example.js

```diff
@@ -39,16 +37,17 @@ const genesis = `{
   "alloc": {}
 }`
 
-const config = {
-  "networkID": networkID, // --networkid / Network identifier (integer, 0=Olympic (disused), 1=Frontier, 2=Morden (disused), 3=Ropsten) (default: 1)
-  "maxPeers": 0, // --maxpeers / Maximum number of network peers (network disabled if set to 0) (default: 25)
-  "genesis": genesis, // genesis.json file
-  "nodeDir": ".private-ethereum", // --datadir / Data directory for the databases and keystore
-  "keyStoreDir": "keystore", // --keystore / Directory for the keystore (default = inside the datadir)
-  "enodes": "enode://XXXX@X[::]:XXXX" // --bootnodes / Comma separated enode URLs for P2P discovery bootstrap
-}
+  const config = {
+    "networkID": networkID, // --networkid / Network identifier (integer, 0=Olympic (disused), 1=Frontier, 2=Morden (disused), 3=Ropsten) (default: 1)
+    "maxPeers": 0, // --maxpeers / Maximum number of network peers (network disabled if set to 0) (default: 25)
+    "genesis": genesis, // genesis.json file
+    "nodeDir": ".private-ethereum", // --datadir / Data directory for the databases and keystore
+    "keyStoreDir": "keystore", // --keystore / Directory for the keystore (default = inside the datadir)
+    "enodes": "enode://XXXX@X[::]:XXXX" // --bootnodes / Comma separated enode URLs for P2P discovery bootstrap
+  }
 
-  const geth = new Geth(config)
+  const geth = new Geth()
+  await get.setConfig(config)
```

Comment:
🙏  Saw your comment on slack, thanks for fixing
https://celo-org.slack.com/archives/CL7BVQPHB/p1597425660048400?thread_ts=1597422341.045500&cid=CL7BVQPHB